lib/3scale/backend/opentelemetry.rb

```diff
@@ -0,0 +1,8 @@
+require 'opentelemetry/sdk'
+require 'opentelemetry/instrumentation/sinatra'
+require 'opentelemetry-exporter-otlp'
+
+OpenTelemetry::SDK.configure do |c|
+  c.service_name = 'backend'
```

Comment:
```suggestion
  c.service_name = '3scale-backend'
```

(if service names starting with a number are allowed)